Rakefile

```diff
@@ -197,7 +197,7 @@ def _flatten_project
     next if Dir["#{dir}/*.tf"].empty?
 
     puts "Working on #{Dir[dir + '/*.tf']}" if debug
-    FileUtils.cp( Dir["#{dir}/*"], tmp_dir)
```

Comment:
This was changed to include files other than .tf so that we can pull pingdom IPs (#115)